.github/workflows/nightly_release.yml

```diff
@@ -1,8 +1,6 @@
 name: Nightly Builds
 on:
-  schedule:
-    # Runs every weekday at 1am
-    - cron: 0 1 * * 1-5
```

Comment:
Shouldn't we retain the cron job schedule?

Reply:
Or will you revert this as soon as the PR checks pass?

Reply:
I will revert once i finish testing.